### PR TITLE
BC1-30-Create a responsive header ( AppLand )

### PR DESCRIPTION
**Pull request by MaxRichard2007**

### CODE REVIEW

**Review by ahmadmohammadirad2006 — CHANGES_REQUESTED** [review]

File: app-land/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            <li class=""><a href="#">features</a></li>
+            <li class=""><a href="#">pricing</a></li>
+            <li class=""><a href="#">testimonials</a></li>
+            <div class="ctaButton">
+              <button>Sign in</button>
+            </div>
+          </ul>
+        </nav>
+      </div>
+    </div>
```
[the remark this answers is not in the document]
Comment (MaxRichard2007) [reply app-land/index.html:65]: I know, I will fix it today

File: app-land/scss/_header.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  @media screen and (max-width: 768px) {
+    .hideOnLaptop {
+      display: block;
+    }
+    .hideOnMobile {
+      display: none;
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (MaxRichard2007) [reply app-land/scss/_header.scss:68]: oh, I forgot to use it, thank you for sharing

File: app-land/scss/_typography.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    border: none;
+    border-radius: 2rem;
+    cursor: pointer;
+
+    &:hover {
+      transition: 0.3s ease-in-out;
+      background: $limeGreen;
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (MaxRichard2007) [reply app-land/scss/_typography.scss:35]: OK, I will fix it today

**Review by ahmadmohammadirad2006 — APPROVED** [review]

File: app-land/scss/_typography.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+body {
+  font-family: "Poppins", sans-serif;
```
Comment (MichaelAndish) [inline app-land/scss/_typography.scss:2]: It should be a variable!

File: shared/scss/components/_button.scss [hunk 5]
```diff
@@ -0,0 +1,15 @@
+.cta__button {
+  &--button {
+    width: 7rem;
+    height: 3rem;
+    background: rgb(084, 234, 084);
```
Comment (MichaelAndish) [inline shared/scss/components/_button.scss:5]: primary color should be a variable!

File: shared/scss/components/_button.scss [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    border: none;
+    border-radius: 2rem;
+    cursor: pointer;
+
+    &:hover {
+      transition: 0.3s ease-in-out;
+      background: rgb(071, 198, 071);
```
Comment (MichaelAndish) [inline shared/scss/components/_button.scss:12]: variable

File: shared/scss/base/_reset.scss [hunk 7]
```diff
@@ -1,3 +1,7 @@
+li {
+  list-style-type: none;
+}
+
```
Comment (MichaelAndish) [inline shared/scss/base/_reset.scss:4]: why you added this?! it'll impact all other projects! 
you can define something like this with class name not just li.

@ahmadmohammadirad2006 I expected to be more careful about approving these kinds of things, please be careful for shared directory

**Review by MichaelAndish — CHANGES_REQUESTED** [review]

File: shared/scss/base/_reset.scss (hunk 7 above)
Comment (ahmadmohammadirad2006) [reply shared/scss/base/_reset.scss:4]: @MichaelAndish I will pay more attention next time

File: app-land/scss/_typography.scss [hunk 8]
```diff
@@ -1,5 +1,5 @@
 body {
-  font-family: "Poppins", sans-serif;
+  font-family: $FontFamily;
```
Comment (MichaelAndish) [inline app-land/scss/_typography.scss:2]: it's not based on code convention, how many times I have to repeat it ?! when do you want to learn it?

File: shared/scss/components/_button.scss [hunk 9]
```diff
@@ -2,14 +2,12 @@
...
-    background: rgb(084, 234, 084);
     border: none;
     border-radius: 2rem;
     cursor: pointer;
 
     &:hover {
       transition: 0.3s ease-in-out;
-      background: rgb(071, 198, 071);
     }
   }
```
Comment (MichaelAndish) [inline shared/scss/components/_button.scss:12]: use the existing version of button

**Review by MichaelAndish — CHANGES_REQUESTED** [review]
Resolve the conflict as well

**Review by MichaelAndish — APPROVED** [review]

**Pull request merged**